CHANGELOG.md

```diff
@@ -8,9 +8,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 
 ### Added
-
 - Added `rollback()` method for `TestKit` allowing to rollback blocks added to
   the testkit blockchain. (#8)
+  - Added `TestKit::create_block_with_transaction()` method. (#13)
```

Comment:
Tiny nitpick: This line still 2 extra spaces at the beginning of the line, which leads to incorrect formatting in the built-in GitHub Markdown viewer (can be seen [here](https://github.com/DarkEld3r/exonum-testkit/blob/504db077aa18e2a91a705cb07701ab3826a6497b/CHANGELOG.md)).